data/tags.yaml

```diff
@@ -2,46 +2,60 @@
 #
 # Each definition needs to define either a query_string or a query_dsl.
 #
-# The create_view is an optional boolean that determines whether a view
-# should be created if there are hits.
+# The save_search is an optional boolean that determines whether a saved
+# search should be created if there are hits.
 #
 # The tagger works in the way that the query is run and the emojis and
 # tags that are defined are applied to the matching events.
-# A view is created if the tag is added (only if there are results).
+#
+# Optionally you can define a regular expression and run it against an
+# attribute that is returned back from the search and only if there is
+# a match against the regular expression the tag will be added.
+#
+# A search is saved if the tag is added (only if there are results).
 #
 # Example:
 #
 # test_tagger:
 #        query_string: 'test'
 #        tags: ['test-tag']
 #        emojis: ['FISHING_POLE']
-#        create_view: true
-#        view_name: 'TEST view name'
+#        save_search: true
+#        search_name: 'TEST the tag'
+#
+# test_tagger_with_re:
+#        query_string: 'test'
+#        tags: ['test-retag']
+#        emojis: ['FISHING_POLE']
+#        save_search: true
+#        regular_expression: '[a-z][A-C]?://foobarX{0,4}'
```

Comment:
Do we want to add re_flags as well? In features.yaml we do support flags:
```
re: '([a-zA-Z0-9_\.+\-]+@[a-zA-Z0-9\-]+\.[a-zA-Z0-9\-\.]+)'
re_flags: []
```

Reply:
sure, SGTM, I'll add that